core/store/orm/config.go

```diff
@@ -792,6 +792,19 @@ func (c Config) LogLevel() LogLevel {
 	return c.getWithFallback("LogLevel", parseLogLevel).(LogLevel)
 }
 
+// ServiceLogLevel is the log level set for a specified package
+func (c Config) ServiceLogLevel(serviceName string) (string, error) {
```

Comment:
I don't think we need this on the config at all right  - just do the DB write as part of SetServiceLogger then we have the db write and in memory write all in one place